zarr/tests/test_storage.py

```diff
@@ -1802,12 +1802,24 @@ def test_context_manager(self):
             store[self.root + 'baz'] = b'qux'
             assert 2 == len(store)
 
+    def test_delitem(self):
+        store = self.create_store()
+        store[self.root + 'foo'] = b'bar'
+
+        del store[self.root + 'foo']
+
+        assert store[self.root + 'foo'] == b''
+
     def test_pop(self):
         # override because not implemented
         store = self.create_store()
         store[self.root + 'foo'] = b'bar'
-        with pytest.raises(NotImplementedError):
-            store.pop(self.root + 'foo')
+
+        store.pop(self.root + 'foo')
+
+        assert store[self.root + 'foo'] == b""
+        # with pytest.raises(NotImplementedError):
+        #     store.pop(self.root + 'foo')
 
     def test_popitem(self):
         # override because not implemented
```

Comment:
We can update this test to run as well (and drop `NotImplementedError` from the test)